ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Block.hs

```diff
@@ -168,14 +171,23 @@ pattern TeleAlonzo  ::
   -> f (ShelleyBlock (TPraos c) (AlonzoEra  c))
   -> Telescope g f (CardanoEras c)
 
+pattern TeleBabbage  ::
+     g ByronBlock
+  -> g (ShelleyBlock (TPraos c) (ShelleyEra c))
+  -> g (ShelleyBlock (TPraos c) (AllegraEra c))
+  -> g (ShelleyBlock (TPraos c) (MaryEra    c))
+  -> g (ShelleyBlock (TPraos c) (AlonzoEra  c))
+  -> f (ShelleyBlock (Praos c)  (BabbageEra c))
+  -> Telescope g f (CardanoEras c)
 -- Here we use layout and adjacency to make it obvious that we haven't
 -- miscounted.
 
-pattern TeleByron                              x =                                            TZ x
-pattern TeleShelley byron                      x = TS byron                                  (TZ x)
-pattern TeleAllegra byron shelley              x = TS byron (TS shelley                      (TZ x))
-pattern TeleMary    byron shelley allegra      x = TS byron (TS shelley (TS allegra          (TZ x)))
-pattern TeleAlonzo  byron shelley allegra mary x = TS byron (TS shelley (TS allegra (TS mary (TZ x))))
+pattern TeleByron                              x        =                                                       TZ x
+pattern TeleShelley byron                      x        = TS byron                                             (TZ x)
+pattern TeleAllegra byron shelley              x        = TS byron (TS shelley                                 (TZ x))
+pattern TeleMary    byron shelley allegra      x        = TS byron (TS shelley (TS allegra                     (TZ x)))
+pattern TeleAlonzo  byron shelley allegra mary x        = TS byron (TS shelley (TS allegra (TS mary            (TZ x))))
```

Comment:
Style: please move the `x`s on the preceding lines' patterns over to line up with the new line's pattern's `x`